docs/resources/enable_data_protection.md

```diff
@@ -38,9 +42,7 @@ resource "tanzu-mission-control_enable_data_protection" "demo" {
 
 ### Required
 
-- `cluster_name` (String) Cluster name
-- `management_cluster_name` (String) Management cluster name
-- `provisioner_name` (String) Cluster provisioner name
+- `scope` (Block List, Min: 1, Max: 1) Scope block for Data Protection (cluster/cluster group) (see [below for nested schema](#nestedblock--scope))
```

Comment:
Do not specify cluster group now. Scope block can accommodate cluster group but do not mention in the schema right now.

Reply:
Done